lms/djangoapps/oauth2_handler/handlers.py

```diff
@@ -190,23 +189,29 @@ def find_courses(self, user, access_type, values=None):
 
         return course_ids
 
-    # pylint: disable=missing-docstring
     def _get_courses_with_access_type(self, user, access_type):
+        """
+        Returns all courses for global staff.  Otherwise, returns list of courses
```

Comment:
Nit: "Returns all courses for global staff" -> "If global staff, returns all courses."